scripts/create-package/plop-templates-react/.babelrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["module:@fluentui/babel-make-styles", "annotate-pure-calls", "@babel/transform-react-pure-annotations"]
```

Comment:
this will add `"module:@fluentui/babel-make-styles"` plugin no matter if it's needed or not. I don't think that's the way we wanna go forward -> one of reasons why I asked to implement this within nx migration generator

Reply:
That would make sense for the entire `create-package` utility to be an nx generator. Generally the packages we create will be for components. Even with an nx generator we would have no way of enforcing that a component package will have the babel config.

This is a good place to have until we can leverage nx fully IMO

Reply:
> That would make sense for the entire create-package 

yeah that's the plan

> Generally the packages we create will be for components. 

thats fine, again we can distinguish accordingly this info in migration generator. In future every package should use `.babelrc.json` no matter for what platform it is.

> Even with an nx generator we would have no way of enforcing that a component package will have the babel config.

hmm not sure I follow ?

We can easily distinguish what plugins should be added for particular converged package or not via nx generator.

Reply:
also this is not blocking, just pointing out what needs to be done (here or another PR) ideally at one place (which is the primary source of truth at the moment - nx migration generator)

Reply:
I meant that it is a sufficient solution to leverage `create-package` so we make sure new packages are created with approriate babel config before an nx generator is ready (more complicated and longer to do)

Reply:
Adding the file to all new react packages seems like a "good enough" approach in the short term (until someone has time to convert create-package to a generator). Most new react packages will be component packages which need this file (if I'm understanding right), and if the file wasn't in the template we'd need another round of these manual fixes. If there's some package where the plugins aren't needed, that can be caught in the PR review.

Reply:
created issue for what was discussed https://github.com/microsoft/fluentui/issues/19109